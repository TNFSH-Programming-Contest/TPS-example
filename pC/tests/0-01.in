12.3 4.56 7.89
