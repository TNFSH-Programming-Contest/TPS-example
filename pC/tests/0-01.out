7.10874525
